constraint/pkg/externaldata/cache_test.go

```diff
@@ -154,7 +153,7 @@ func TestRemove(t *testing.T) {
 	tc := []cacheTestCase{
 		{
 			Name:          "valid provider",
-			Provider:      createProvider("test", "http://test", 1, "", true),
+			Provider:      createProvider("test", "http://test", 1, ""),
```

Comment:
should http be valid?

Reply:
no http is not valid, https only

Reply:
good catch, this is a remove test. I'll update to https